.github/workflows/main.yml

```diff
@@ -113,3 +113,15 @@ jobs:
         pip install tox virtualenv
     - name: Test build
       run: "tox -e ${{ matrix.tox_env }}"
+    - name: Send coverage report to codeclimate
+      uses: paambaati/codeclimate-action@v2.6.0
+      with:
+        coverageCommand: echo "Ignore rerun"
+        coverageLocations: ${{github.workspace}}/coverage.xml:coverage.py
+      env:
+        CC_TEST_REPORTER_ID: ${{secrets.CC_TEST_REPORTER_ID}}
```

Comment:
We should open an issue to expose `CC_TEST_REPORTER_ID` as an env and get this stage on the pull request.

By default, secrets are not exposed in the pull requests ci.

Otherwise is pretty useless.

Reply:
#1523